common/test/test_microsites/test_microsite/templates/static_templates/about.html

```diff
@@ -19,7 +19,7 @@
     ##   <div class="logo">
     ##     <img src="${static.url('images/edx-logo-large-bw.png')}">
     ##   </div>
-    ##   <h2 class="mission-quote">&ldquo;${_('The mission of <span class="edx">{edX}</span> is to enhance human fulfillment worldwide through online ## learning, transforming education in quality, efficiency and scale through technology and research, for the benefit of campus-based ## students and the worldwide community of online learners.').format(edX="edX")}&rdquo;</h2>
+    ##   <h2 class="mission-quote">&ldquo;${_('The mission of {edX} is to enhance human fulfillment worldwide through online ## learning, transforming education in quality, efficiency and scale through technology and research, for the benefit of campus-based ## students and the worldwide community of online learners.').format(edX='<span class="edx">edX</span>')}&rdquo;</h2>
```

Comment:
These changes are in test files. The better solution is to de-i18n all the strings in this file since it's a test file.